### ISSUE TITLE
Last 3 dev builds.  fatal signal 11 on raspberry pi

### ISSUE THREAD

**Issue by sfeakes** [domoticz/domoticz#1605]
The last 3 dev builds have caused a fatal crash after 2~4 hours of running on raspberry pi 2+
No changes to system except the dev builds, Below are the only errors seen in the log.

Latest version :-
Version: 3.7698
Build Hash: d6d1b29

Error: Domoticz received fatal signal 11 !...
Error: Domoticz received fatal signal 11 while backtracing !...
Error:   /home/domoticz/domoticz() [0x1cc7f4]
Error:   /home/domoticz/domoticz(_Z14signal_handleri+0x58) [0x1cc8ac]
Error:   /lib/arm-linux-gnueabihf/libc.so.6(__default_sa_restorer_v2+0) [0x76a59160]

**Comment by mikeo999** [comment]
I have the same problem. Domoticz keeps crashing random a few times a day now. 
Every time it takes a while before Monit gets Domoticz running again. Then it run again for a few hours and crash again.

Before the last few beta updates it was running without any problem.

2017-06-10 07:33:04.852  Error: (Buienradar) failed to create interpreter.
2017-06-10 07:33:04.867  Error: Domoticz received fatal signal 11 !...
2017-06-10 07:33:04.881  Error:   /home/pi/domoticz/domoticz() [0x1cc7f4]
2017-06-10 07:33:04.881  Error:   /home/pi/domoticz/domoticz(_Z14signal_handleri+0x58) [0x1cc8ac]
2017-06-10 07:33:04.882  Error:   /lib/arm-linux-gnueabihf/libc.so.6(__default_sa_restorer_v2+0) [0x76b80b20]

**Comment by gizmocuz** [comment]
Do you have a file 'script_device_PIRsmarter.py' in your scripts/python folder ? If yes, could you remove it
If you can build domoticz with debugging symbols a better crash report could be displayed

**Comment by mikeo999** [comment]
I removed script_device_PIRsmarter.py and restart the system.  Let you know if the system is still crashing, but is random. Mostly late in the evening it crash again.

I can't rebuild domoticz because i have a lot of stuff installed in the past month and hard to recreate everything. . So hoping we can solve the problem without doing that.

**Comment by gizmocuz** [comment]
That is building in a new folder
Copy all except the executable to the new build folder and run from command line 
Remember to NOT build in release mode

**Comment by sfeakes** [comment]
I noticed on machine boot-up Domoticz wouldn't start from the init.d script, but it would start correctly (then crash after a few hours) when running the init.d script directly.  When it failed on machine boot, there was always a "plugin subsystem" message just before the errors listed above.  So I've disabled all the plugin's and it's run for 12 hours now with no crashing.  If it continues to run I'll add the plugin's back one at a time to see which one causes the failure.
I also notice there is a new build, I may try that but don't want to add too many unknowns at the moment.  If I get chance, I'll compile Domoticz with debug enabled and try that.
